apps/annotations/models.py

```diff
@@ -22,12 +26,14 @@
 )
 class Tag(TagBase, BaseTeamModel):
     name = models.CharField(verbose_name=pgettext_lazy("A tag name", "name"), max_length=100)
-    created_by = models.ForeignKey("users.CustomUser", on_delete=models.DO_NOTHING)
+    created_by = models.ForeignKey("users.CustomUser", on_delete=models.DO_NOTHING, null=True, default=None)
+    is_system_tag = models.BooleanField(default=False)
+    category = models.CharField(choices=TagCategories.choices, default="bot_response")
```

Comment:
I think the default here should be blank otherwise all user generated tags will have this default value.
```suggestion
    category = models.CharField(choices=TagCategories.choices, blank=True, default="")
```